File: testdata/bench/c/BenchmarkTest00001.c
#include <stdio.h>
#include <sqlite3.h>

void handle_request(const char *user_input) {
    char query[256];
    sqlite3 *db;
    sqlite3_open(":memory:", &db);
    sprintf(query, "SELECT * FROM users WHERE id = %s", user_input);
    sqlite3_exec(db, query, NULL, NULL, NULL);
    sqlite3_close(db);
}

File: testdata/bench/swift/BenchmarkTest00001.swift
import Vapor
import SQLite3

func handler001(_ req: Request) throws -> String {
    let userId = try req.parameters.require("id")
    let query = "SELECT * FROM users WHERE id = '\(userId)'"
    var db: OpaquePointer?
    sqlite3_open(":memory:", &db)
    sqlite3_exec(db, query, nil, nil, nil)
    return "ok"
}
